datasets/wit/README.md

```diff
@@ -0,0 +1,346 @@
+total > 100K | 36     | images > 100K | 35
+total > 50K  | 15     | images > 50K  | 17
+total > 14K  | 38     | images > 13K  | 38
+
+## Dataset Structure
+
+### Data Instances
+
+```
+{
+  'language': 'en',
+  'page_url': 'https://en.wikipedia.org/wiki/Oxydactylus',
+  'image_url': 'https://upload.wikimedia.org/wikipedia/commons/5/5f/Oxydactylus_longipes_fm.jpg',
+  'page_title': 'Oxydactylus',
+  'section_title': None,
+  'hierarchical_section_title': 'Oxydactylus',
+  'caption_reference_description': None,
+  'caption_attribution_description': 'English: Mounted skeleton of Oxydactylus longipes in the Field Museum of Natural History.',
+  'caption_alt_text_description': None,
+  'mime_type': 'image/jpeg',
+  'original_height': 3564,
+  'original_width': 2748,
+  'is_main_image': True,
+  'attribution_passes_lang_id': True,
+  'page_changed_recently': True,
+  'context_page_description': 'Oxydactylus is an extinct genus of camelid endemic to North America. It lived from the Late Oligocene to the Middle Miocene, existing for approximately 14 million years. The name is from the Ancient Greek οξύς and δάκτυλος.\nThey had very long legs and necks, and were probably adapted to eating high vegetation, much like modern giraffes. Unlike modern camelids, they had hooves, rather than tough sole-pads, and splayed toes.',
+  'context_section_description': 'Oxydactylus is an extinct genus of camelid endemic to North America. It lived from the Late Oligocene to the Middle Miocene (28.4–13.7 mya), existing for approximately 14 million years. The name is from the Ancient Greek οξύς (oxys, "sharp")and δάκτυλος (daktylos, "finger").\n \nThey had very long legs and necks, and were probably adapted to eating high vegetation, much like modern giraffes. Unlike modern camelids, they had hooves, rather than tough sole-pads, and splayed toes.'
+}
+```
+
+### Data Fields
+
+- `language`: Language code depicting wikipedia language of the page
+- `page_url`: URL to wikipedia page
+- `image_url`: URL to wikipedia image
+- `page_title`: Wikipedia page's title
+- `section_title`: Section's title
+- `hierarchical_section_title`: Hierarchical section's title
+- `caption_reference_description`: This is the caption that is visible on the wiki page directly below the image.
+- `caption_attribution_description`: This is the text found on the Wikimedia page of the image. This text is common to all occurrences of that image across all Wikipedias and thus can be in a language different to the original page article.
+- `caption_alt_text_description`: This is the “alt” text associated with the image. While not visible in general, it is commonly used for accessibility / screen readers
+- `mime_type`: Mime type associated to the image.
+- `original_height`: Image height
+- `original_width`: Image width
+- `is_main_image`: [More Information Needed]
+- `attribution_passes_lang_id`: [More Information Needed]
+- `page_changed_recently`: [More Information Needed] 
```

Comment:
I was unclear on those. My guess is:
 - `is_main_image`: probably the first image, typically the one that displays on the right in the web browser at the top.
 - `attribution_passes_lang_id`: compares the `language` field with the attribution language (written in the prefix of the attribution description)
 - `page_changed_recently`: ???
 
 I don't know if I want to fill them with guesses and I couldn't find documentation ... Seeing as this will live inside `google/wit` I would like to defer documentation to authors ...

Reply:
Maybe @yjernite can help us with this.

Reply:
Yes on both of @thomasw21 's guesses, `page_changed_recently` is just that, how long since the page where the image and captions were found has been changed in the last X days (don't know X)

Definitely feel free to merge with that

Reply:
Okay so adding the first two descriptions. Leaving the `page_changed_recently` with [More Information Needed] as without knowing X, this field makes little sense to me.